playbooks/templates/rax-maas/ironic_capacity_check.yaml.j2

```diff
@@ -0,0 +1,20 @@
+{% set label = "ironic_api_local_check" %}
+{% set check_name = label+'--'+inventory_hostname %}
+type        : agent.plugin
+label       : "{{ check_name }}"
+period      : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout     : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled    : "{{ (check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
+details     :
+    file    : run_plugin_in_venv.sh
+    args    : ["{{ maas_plugin_dir }}/ironic_api_local_check.py", "{{ ansible_host }}"]
+alarms      :
+    ironic_capacity :
+        label                   : ironic_capacity--{{ inventory_hostname }}
+        notification_plan_id    : "{{ maas_notification_plan_override[label] | default(maas_notification_plan) }}"
+        disabled                : {{ (('ironic_capacity--'+inventory_hostname) | match(maas_excluded_alarms_regex)) | ternary('true', 'false') }}
+        criteria                : |
+            :set consecutiveCount={{ maas_alarm_local_consecutive_count }}
+            if (metric["ironic_capacity"] <= 20) {
```

Comment:
Here's what I was referring to earlier with letting the check calculate the percentage itself. [This check](https://github.com/rcbops/rpc-maas/blob/master/playbooks/templates/rax-maas/conntrack_count.yaml.j2#L18) is an example of what I was talking about.